home.css

```diff
@@ -0,0 +1,561 @@
+
+/* intro section */
+.intro {
+  border: 5px #fff solid;
+  margin-left: 5%;
+  font-size: 110%;
+  margin-right: 15%;
+  margin-top: 5%;
+  font-weight: normal;
+  opacity: 0.7;
+}
+
+.due-date {
+  position: relative;
+  font-weight: bold;
+  color: #585858;
+  font-size: 200%;
+  margin-left: 5%;
+  margin-bottom: 1%;
+  margin-top: 5%;
+}
+
+.venue {
+  color: #222121;
+  margin-left: 5%;
+  font-size: 110%;
+  margin-bottom: 5%;
+  opacity: 0.7;
+}
+
+/* Main program section */
+.program {
+  background: url("./images/main-program.jpg") no-repeat 0, 0 #0a0a0a;
+  background-size: cover;
+  margin-top: 0%;
+}
+
+.main-program {
+  position: relative;
+  color: #fff;
+  padding-top: 10%;
+  padding-bottom: 2%;
+  text-align: center;
+}
+
+.main-program::after {
+  content: '';
+  height: 2px;
+  width: 5%;
+  background: #f35826;
+  position: absolute;
+  bottom: 0;
+  left: 47%;
+}
+
+.activities {
+  padding-top: 5%;
+  list-style-type: none;
+  font-size: 90%;
+  opacity: 0.65;
+}
+
+.activity {
+  display: flex;
+  border: none;
+  background-color: #3c3f46;
+  gap: 10px;
+  margin: 2%;
+}
+
+.activities .activity:hover {
+  cursor: pointer;
+  position: relative;
+  border: #fff 2px solid;
+}
+
+.img-activity {
+  padding: 3%;
+  height: 120px;
+  margin-top: 3%;
+  width: 15%;
+}
+
+.description {
+  display: flex;
+  align-items: center;
+  color: #f35826;
+  font-weight: bold;
+  padding-right: 5%;
+  margin-right: 50px;
+  width: 10%;
+}
+
+.explanation {
+  font-size: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+}
+
+.whole-program {
+  display: none;
+}
+
+.program-btn {
+  position: relative;
+  display: flex;
+  justify-content: center;
+}
+
+.join {
+  padding: 4%;
+  background-color: #f35826;
+  border: none;
+  color: #fff;
+  font-size: 140%;
+  margin: 5%;
+  margin-bottom: 10%;
+}
+
+.join:hover {
+  cursor: pointer;
+}
+
+.join:hover:hover {
+  background-color: #fff;
+  color: #f35826;
+}
+
+.join:hover:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+/* Featured speakers section */
+.speakers-section {
+  margin-top: 5%;
+}
+
+.featured-speakers {
+  text-align: center;
+  position: relative;
+  padding-bottom: 2%;
+}
+
+.featured-speakers::after {
+  content: '';
+  height: 2px;
+  width: 5%;
+  background: #f35826;
+  position: absolute;
+  bottom: 0;
+  left: 47%;
+}
+
+.speakers {
+  list-style-type: none;
+  margin-top: 5%;
+}
+
+.speakers li {
+  display: flex;
+}
+
+.speaker-img {
+  width: 100px;
+  height: 100px;
+}
+
+.speaker-details {
+  padding-left: 3%;
+}
+
+.name {
+  font-weight: bold;
+  padding-bottom: 3%;
+  padding-top: 2%;
+}
+
+.work {
+  color: #f35826;
+  font-style: italic;
+  padding-bottom: 2%;
+}
+
+.div-item {
+  height: 2px;
+  width: 5%;
+  background: grey;
+  opacity: 0.4;
+  margin-bottom: 3%;
+}
+
+.work_description {
+  font-size: 95%;
+}
+
+.partner-section {
+  display: none;
+}
+
+.desktop-footer {
+  display: none;
+}
```

Comment:
While I understand you were following the template design, it is a project requirement to have a footer section in both versions of both the `Home` and `About` pages. Please add it to the mobile version of your `Home` page. I apologize for the confusion.

<img width="316" alt="Screen Shot 2022-06-22 at 9 01 01" src="https://user-images.githubusercontent.com/82426682/175048260-ea4bf351-19b1-4e84-9a88-8680709f35de.png">